src/Params/SUBnoteParameters.cpp

```diff
@@ -38,7 +38,7 @@ static const rtosc::Ports SUBnotePorts = {
     rSelf(SUBnoteParameters),
     rPaste,
     rToggle(Pstereo,    rShort("stereo"), rDefault(true), "Stereo Enable"),
-    rParamZyn(PVolume,  rShort("volume"), rDefault(96), "Volume"),
+    rParamF(Volume,  rShort("volume"), rDefault(0), rLinear(-60.0f,20.0f), "Volume"),
```

Comment:
Could you add units similar to https://github.com/zynaddsubfx/zynaddsubfx/commit/2c035fd23eb21ddde971daeb56af988841f761ff#diff-c3d6d0e24a7c539711c03581b5444362R59 it should just be rUnit(dB).